main/openrc/modloop.initd

```diff
@@ -10,7 +10,7 @@ depend() {
 
 # read kernel options
 init_KOPT() {
-	eval set -- $(cat /proc/cmdline 2>/dev/null)
+	eval set -- $(cat /proc/cmdline 2>/dev/null|sed -e 's/(loop)//g')
```

Comment:
i think we can drop the `eval` yes. It was originally so we would handle options with spaces like `acpi_osi="!Windows 2006"`